src/ralph/models/selector.py

```diff
@@ -152,3 +161,55 @@ def get_decision_tree(self, model_rules):
         false_subtree = self.get_decision_tree(false_subtree) if false_subtree else None
 
         return {root_rule: {True: true_subtree, False: false_subtree}}
+
+    def validate(self, input_file: TextIO, ignore_errors: bool, fail_on_unknown: bool):
```

Comment:
From a software architecture perspective, I think validation should not be bound to the model selector. I would prefer having a separated `EventValidator` that is only responsible for validation of a given model type that is injected as a dependency. WDYT?

Reply:
Great Idea) Thanks) This will also make the classes / tests more manageable)